.envrc

```diff
@@ -0,0 +1 @@
+eval "$(lorri direnv)"
```

Comment:
I would not submit .envrc. That leaves it to each individual how they want their environment to work (e.g. I use nix-direnv and `use nix` instead of lorri, for reasons). That's also what I did in the corp repo, shell.nix is there if you want it but it's up to you how you enable it in your environment.